pkgs/development/python-modules/treescope/default.nix

```diff
@@ -0,0 +1,106 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  flit-core,
+
+  # dependencies
+  numpy,
+
+  # optional-dependencies
+  ipython,
+  jupyter,
+  # pyink,
+  pylint,
+  jax,
+  matplotlib,
+  myst-nb,
+  myst-parser,
+  palettable,
+  pandas,
+  # penzai,
+  plotly,
+  sphinx,
+  sphinx-book-theme,
+  # sphinx-contributors,
+  sphinx-hoverxref,
+  sphinxcontrib-katex,
+  torch,
+  absl-py,
+  pytest,
+
+  # tests
+  jaxlib,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "treescope";
+  version = "0.1.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "google-deepmind";
+    repo = "treescope";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+Hm60O9tEXIiE0av1O0BsOdMln4e1s7ijb3WNiQ74jE=";
+  };
+
+  build-system = [ flit-core ];
+
+  dependencies = [ numpy ];
+
+  optional-dependencies = {
```

Comment:
most of these are not needed for distribution. Please drop all but `notebook`

Reply:
Done !